### PR TITLE
Fix test with address list

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
Wait: there is one more test failing!
Fixed the test failing after merging the original PR.
I think it was a mistake to  use the same branch, but it is what is for now. I have commented the changes that actually fix the test.
And this the one new commit:
249b10e09151aa66995d9896a1efa556d105b5af

### CODE REVIEW

File: ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.spec.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ class FakeAddressInputComponent {
   private _readonly = false;
   @Output()valueChanged = new EventEmitter();
   @Input()addressErrors;
-  @Input()country;
+  @Input()country = null;
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.spec.ts:33]: This is one change for the test. Test was expecting null but we were getting undefined.

File: ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.spec.ts [hunk 2]
```diff
@@ -476,14 +476,7 @@ describe('AddressEmbeddedComponent', () => {
...
 
-    expect(childComponent.country).toBeNull();
-    component.activityGuid = '123';
-    component.country = 'CA';
-    tempAddress = new Address({country: 'CA'});
-    addressServiceSpy.getTempAddress.and.returnValue(of(tempAddress));
-    component.ngOnInit();
-    fixture.detectChanges();
-    expect(childComponent.country).toBe('CA');
+    expect(childComponent.country).toBe('US');
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.spec.ts:479]: I changed behavior. I was originally allowing for country in input form to show up if we get a full address that had a country that was different from the default country. For the sake of simplicity, I dropped that capability, so this chunk of test was no longer valid.

**Comment by fat23cat** [comment]
Tip: to prevent accidental merge, you can convert the PR to draft in addition to the 'Draft' label:
image

**Comment by MocanaAtBroad** [comment]
> Tip: to prevent accidental merge, you can convert the PR to draft in addition to the 'Draft' label:
> image

I can never find that link! Thanks! (At any rate, I think it is good now).

**Review by fat23cat — APPROVED** [review]

**Pull request merged**